src/home/views.py

```diff
@@ -99,14 +113,49 @@ def get(self):
     def post(self):
         data = flask.request.get_json()
         email = data.get('email')
+        securityCode = data.get('securityCode')
         if email:
             user = models.User.query.filter(models.User.email.contains(email)).first()
             if user:
-                user.permissions = 3
-                models.db.session.commit()
-                return email
+                if user.permissions:
```

Comment:
I was going to leave it this way for testing purposes and then we can change it later